app/services/application_state_change.rb

```diff
@@ -122,6 +122,10 @@ def persist_workflow_state(new_state)
     update_candidate_api_updated_at_if_application_forms_state_has_changed(previous_application_form_status, current_application_form_status)
   end
 
+  def self.reapply_states
```

Comment:
Is this only being used in the spec files?

Reply:
Ah yes, I was meant to use it in the ApplicationChoice scope